### PR TITLE
 Set up v0 API endpoint for program listing

### PR DESCRIPTION
**Pull request by kdmccormick**
[EDUCATOR-4171](https://openedx.atlassian.net/browse/EDUCATOR-4171)
@edx/masters-neem

### CODE REVIEW

**Comment by zacharis278** [comment]
It would be useful to split our mock data into some util.  IMO this view is going to become difficult to navigate if we put our mocks alongside the view logic.

**Comment by kdmccormick** [comment]
@Zacharis278 Good thinking. I pushed a change that breaks it out in to `data.py` and structures it a little better. Let me know what you think.

File: registrar/apps/api/v0/data.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        program_key,
+        org,
+        title,
+        'https://{}.edx.org/{}'.format(org.key, program_key),
+    )
+
+
+# List of fake organizations
+FAKE_ORGS = [
+    FakeOrganization('u-perezburgh', 'UPZx', False, False, False),
```
Comment (jansenk) [inline registrar/apps/api/v0/data.py:38]: Love these names 👍

Comment (kdmccormick) [reply registrar/apps/api/v0/data.py:38]: Thanks XD Got them with help from the Faker library.

File: registrar/apps/api/v0/views.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    FAKE_ORG_PROGRAMS,
+    FAKE_PROGRAM_DICT,
+)
+
+
+class MockProgramListView(ListAPIView):
+    """
+    A view for listing program objects.
+
+    Path: /api/v1/programs?org={org_key}
```
Comment (zacharis278) [inline registrar/apps/api/v0/views.py:25]: v0.  ditto for the other view

File: registrar/apps/api/v0/views.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+    All programs within organization specified by `org_key` are returned.
+    For users will global organization access, `org_key` can be omitted in order
```
Comment (zacharis278) [inline registrar/apps/api/v0/views.py:28]: sp. will/with

**Review by zacharis278 — APPROVED** [review]
Found two minor doc typos.  Otherwise this looks good

**Pull request merged**